Add YouTube Link to Footer

chrislarrycarl opened the pull request:
## Description:
Add youtube to footer social menu, this change has already been pushed to the Drupal production site

### Resolves:
* Request in Asana

alexdahl-okta commented:
Is there an icon for Youtube? Looks broken on the marketing side.

chrislarrycarl commented:
@alexdahl-okta There is in the Drupal CSS, which also was cached but has now been cleared. Try clearing it on your end again and it should display now. Thanks.

alexdahl-okta commented:
@chrislarrycarl 👍 Looks like it just needed some time to expire, thanks! 

And we don't need to make any css changes on this side? (I see declarations for each of the existing social links in [_Footer.scss](https://github.com/okta/okta.github.io/blob/source/_source/_assets/css/okta/components/_Footer.scss), for example.)

Sorry for questions, just caught my eye!

chrislarrycarl commented:
@alexdahl-okta The shared styles (header & footer) are driven by the Drupal stylesheets so there shouldn't be any css changes needed for this change. However, looking at the non-Drupal footer more closely it looks like the social nav spacing is off. I've adjusted the spacing to bring it more inline with the Drupal footer, added youtube for consistency, but removed spacing from the last item in the list to keep the footer tightened up. Thanks!

alexdahl-okta commented:
@chrislarrycarl  Thanks! 👏 lgtm, but should probably get another approval.
